### DISCUSSION TITLE
Booting from SSD vs. SD Card?

### DISCUSSION THREAD

**Discussion by PtJudeRI** [bareboat-necessities/lysmarine_gen#406] [Q&A]
What is the consensus on Running your system on an SSD vs. an SD card?  I like the compact nature of my RPi as it is with the SD card, but have heard that over time, I should expect corruption issues with the SD card.  Not so with an SSD, or much lower chance of corruption.    Would an SSD boot be a more robust option?  Can I keep a cloned SD card handy in case something goes pear shaped?

**Comment by mgrouch** [comment]
Yes SSD is better and faster. 
You can keep SD card to restore the system.
